### DISCUSSION TITLE
[@types/accepts] Broken with ESM

### DISCUSSION THREAD

**Discussion by evert** [DefinitelyTyped/DefinitelyTyped#64233] [Issues with a @types package]
The 'accepts' package has a default export that's both a constructor and a function.

None of these syntaxes work (with `allowSyntheticDefaultImports`):

```typescript
 import * as accepts from 'accepts';
 const foo = accepts('..');
```

During runtime: `TypeError: accepts is not a function`

```typescript
 import accepts from 'accepts';
 const foo = accepts('..');
```

During compile time: `This module is declared with 'export =', and can only be used with a default import when using the 'allowSyntheticDefaultImports' flag.`

I'm not sure how I could use this module. `allowSyntheticDefaultImports` is not an option for me, because this is a library that should support true or false values downstream for this flag.

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment]
Does this syntax work?

```typescript
import accepts = require('accepts');
const foo = accepts('..');
```

**Reply by evert** [reply]
Unfortunately, `require` does not work with ESM. (ES6 modules).
